server/test/unit/server/webservices/test_urls.py

```diff
@@ -3,6 +3,40 @@
 from django.core.urlresolvers import resolve, Resolver404
 
 
+class TestDjangoUrls(unittest.TestCase):
+    """
+    Base class for testing Django Urls.
+    """
+    def assert_url_match(self, urls, url_name, match=True, *args):
+        """
+        Asserts that a url matches the url_name if match == True,
+        asserts that a url does not match the url_name if match==False.
+
+        :param url     : url or urls to match or base url to build from
+        :type  url     : string or list of strings
+        :param url_name: expected name of the url match
+        :type  url_name: string
+        :param match   : If true, check that urls match, if false, check that urls do not match
+        :args          : optional arguments to append to the end of the url
```

Comment:
:args is not a recognized sphinx command. I think :param args maybe is what you want.

:args is also missing a :type describing its type